kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/test/TestCaseExecutor.kt

```diff
@@ -86,7 +85,6 @@ internal class TestCaseExecutor(
             timeMark,
             listOfNotNull(
                InvocationTimeoutInterceptor,
-               if (platform == Platform.JVM && testCase.config.testCoroutineDispatcher) createTestDispatcherInterceptor() else null,
                if (useCoroutineTestScope) TestCoroutineInterceptor() else null,
```

Comment:
If I understood the changes in #4248 correctly, you'd prefer Kotest to throw an exception in case a configured test dispatcher meets a lack of platform support. While probably just a temporary situation, I think this makes sense. So the above code

``` kotlin
      val useCoroutineTestScope = when (platform) {
         Platform.JVM, Platform.Native -> testCase.config.coroutineTestScope
         Platform.JS, Platform.WasmJs -> false
      }
```

could be changed to
``` kotlin
      val useCoroutineTestScope = when (platform) {
         Platform.JVM, Platform.Native -> testCase.config.coroutineTestScope
         Platform.JS, Platform.WasmJs -> if (testCase.config.coroutineTestScope) {
            error("Configuration 'coroutineTestScope' is unsupported on $platform")
         } else false
      }
```

Reply:
done